localization/english/ancient_magic_custom_description_l_english.yml

```diff
@@ -1,5 +0,0 @@
-﻿l_english:
```

Comment:
why was this deleted ? The reason there is this loc, is becaus evanilla doesnt has or, didnt have it before, dont know about now